video-player-in-flutter-using-chewie-and-getx/index.md

```diff
@@ -0,0 +1,287 @@
+```dart
+Widget buildButton(String title, BuildContext context) { 
+ return new Container( 
+    height: 50.0, padding: const EdgeInsets.only(top: 5.0),
+     decoration: new BoxDecoration( color: Colors.white, ),
+      child: new Text( title, style: const TextStyle(fontWeight: FontWeight.bold, fontSize: 20.0), 
+      ), 
+      ); 
+      }
+```
+We use a `Container` to create our button with a height of 50dp and top padding of 5dp.
+
+Next, we add a `BoxDecoration` to apply a white background with the same height and width as our `Container`, and finally, we get the text inside the button. In this case, it is just some random string I typed in, but you usually use an `IconButton` to get the correct icon.
+
+Finally, we use the `buildButton` method that receives as a parameter the string that should be inside our button, and then it returns the built button `Container`.
+
+We do almost the same thing to build the other button:
+
+```dart
+
+Widget buildButton(String title, BuildContext context) { 
+
+ return new Container(
+
+    height: 50.0, padding: const EdgeInsets.only(top: 5.0), decoration: new BoxDecoration( 
+
+      color: Colors.white, 
+
+      ), 
+
+      child: new Text( title, style: const TextStyle(fontWeight: FontWeight.bold, fontSize: 20.0), 
+
+      ),
+
+ ); 
+
+}
+
+```
+
+This time, we only have to modify the text a little bit. First, the "STEP_BACKWARD" string should appear in blue, so we add a second parameter called `color` with the value `Colors. blue`.The purpose of this is to help you better understand the components without complex implementation of the same functions. 
+
+Almost done! We have to update our main method to show the video player:
+
+```dart
+
+void main() { 
+
+ runApp(new VideoPlayerApp());
+
+}
+
+```
+
+We will instantiate the `VideoPlayerApp` in the main method, and then we create a new instance of it (using our video URL) to start playing.
+The RunApp method runs our application. For this case it will run the VideoPlayerApp.
+
+Widgets used in this example:
+
+```dart
+
+Widget buildButton(String title, BuildContext context) { 
+
+ return new Container( 
+
+   height: 50.0, padding: const EdgeInsets.only(top: 5.0), decoration: new BoxDecoration( 
+
+     color: Colors.white, ),
+
+      child: new Text( title, style: const TextStyle(fontWeight: FontWeight.bold, fontSize: 20.0),
+
+   ), 
+
+ ); 
+
+}
+
+Widget buildButton(String title, BuildContext context, [Color color]) { 
+
+ return new Container(
+
+   height: 50.0, padding: const EdgeInsets.only(top: 5.0), decoration: new BoxDecoration( color: color, ),
+
+    child: new Text( title,
+
+     style: const TextStyle(fontWeight: FontWeight.bold, fontSize: 20.0),
+
+    ), 
+
+ ); 
+
+}
+
+```
+ Chewie + Getx is a very good combination to make an app like this in Flutter.You can test the appliaction on your phone and add more features to it and use it for your own good or use to develop your own videoplayer.
```

Comment:
Change to "  Chewie and Getx is a perfect combination to make an app like this in Flutter. You can test the application on your phone, add more features, and use it for your good or develop your video player."